src/sass/player.sass

```diff
@@ -7,13 +7,13 @@
         transform: rotate(360deg)
 
 .penguin-player
-    font-family: "Microsoft YaHei UI", "Microsoft YaHei", "黑体"
+    font-family: -apple-system, BlinkMacSystemFont, 'Noto Sans', 'Source Han Sans', 'Microsoft YaHei UI', 'Microsoft YaHei', 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
```

Comment:
Have you tested the player with all of these fonts?

Reply:
yup but -apple-system and BlinkMac

Reply:
This is what vscode tell me to do

Reply:
but it should be no problem when running on macOS.

Reply:
Well, then I will test it myself later